### ISSUE TITLE
negative resize flicker bug

### ISSUE THREAD

**Issue by dwelle** [excalidraw/excalidraw#1443]
I believe this is a new regression. Will need to bisect to find the commit that introduced it.

![excal_negative_resize_bug](https://user-images.githubusercontent.com/5153846/79485751-09c89e80-8016-11ea-96e0-2f49c1a1d393.gif)

/cc @dai-shi

**Comment by dai-shi** [comment]
Most likely, it's from our refactoring in https://github.com/excalidraw/excalidraw/pull/1330#issuecomment-611585017.
We don't want to revert it, but we want to fix it.

Lately, I've been working on #1427 and I'd say this is more difficult with convex curved lines.
I'd extract some bugfix work from #1427 and create a separate PR, once #1225 is merged.

**Comment by dwelle** [comment]
Yep, that seems to be it. In particular, this commit: https://github.com/excalidraw/excalidraw/pull/1330/commits/8330966a2d62eb61fae6a5401ee3beb5256ce34f, if it proves useful later.

**Comment by dai-shi** [comment]
In addition to https://github.com/excalidraw/excalidraw/pull/1475#issuecomment-618885039, this behavior is still not comfortable for me. Should we open a new issue or keep this?

![exdraw31](https://user-images.githubusercontent.com/490574/80439215-efa98d00-8940-11ea-9fd7-a32d8101bc61.gif)

**Comment by dwelle** [comment]
I've created 2 new issues:

- [#1505 Resizing elements should offset initial cursor delta](https://github.com/excalidraw/excalidraw/issues/1505)
- [#1506 Negative resizing should be continuous](https://github.com/excalidraw/excalidraw/issues/1506)

and added them to the epic.

Let's close this one so we potentially refer back to it in case of regressions etc.

**Comment by dai-shi** [comment]
Thanks!
